PointCloud2: Huge point size when window is maximized

looks like in the noetic rviz release there seems to be a bug when you try to change the point size for pointcloud2 topics.

Here is a screenshot of the default pointcloud points that show up when I add in the viz, and yes I have tried to change the type/size of it:
![pointcloud_points](https://user-images.githubusercontent.com/6157095/82958697-ce9d8180-9f6a-11ea-89b1-6adab2ddb601.png)


### Your environment
* OS Version: e.g. Ubuntu 20.04
* ROS Distro: Noetic
* RViz, Qt, OGRE, OpenGl version as printed by rviz:

```
[ INFO] [1590534094.203451371]: rviz version 1.14.0
[ INFO] [1590534094.203502146]: compiled against Qt version 5.12.8
[ INFO] [1590534094.203514196]: compiled against OGRE version 1.9.0 (Ghadamon)
[ INFO] [1590534094.212776806]: Forcing OpenGl version 0.
[ INFO] [1590534094.416371615]: Stereo is NOT SUPPORTED
[ INFO] [1590534094.416444253]: OpenGl version: 4.6 (GLSL 4.6).
```

After compilation with ogre 1.12:

> $rosrun rviz rviz 
> [ INFO] [1590780380.302481058]: rviz version 1.14.0
> [ INFO] [1590780380.302609856]: compiled against Qt version 5.12.8
> [ INFO] [1590780380.302650942]: compiled against OGRE version 1.12.4 (Rhagorthua)
> [ INFO] [1590780380.325357004]: Forcing OpenGl version 0.
> [ INFO] [1590780380.626125246]: Stereo is NOT SUPPORTED
> [ INFO] [1590780380.626204835]: OpenGl version: 4.6 (GLSL 4.6).
> [ERROR] [1590780380.639836601]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
> 
Rviz starts, then crash when trying to display a pointcloud:

> [ERROR] [1590780429.109706877]: No techniques available for material [PointCloudMaterial0Sphere]
> [ERROR] [1590780429.110194420]: No techniques available for material [PointCloudMaterial0FlatSquare]
> [ERROR] [1590780429.619428720]: No techniques available for material [PointCloudMaterial1Sphere]
> [ERROR] [1590780429.619645848]: No techniques available for material [PointCloudMaterial1FlatSquare]
> [ERROR] [1590780430.098322011]: No techniques available for material [PointCloudMaterial2Sphere]
> [ERROR] [1590780430.098555597]: No techniques available for material [PointCloudMaterial2FlatSquare]
> [ERROR] [1590780430.608390163]: No techniques available for material [PointCloudMaterial3Sphere]
> [ERROR] [1590780430.608440268]: No techniques available for material [PointCloudMaterial3FlatSquare]
> [ERROR] [1590780431.089843225]: No techniques available for material [PointCloudMaterial4Sphere]
> [ERROR] [1590780431.089972989]: No techniques available for material [PointCloudMaterial4FlatSquare]
> [ERROR] [1590780431.601881442]: No techniques available for material [PointCloudMaterial5Sphere]
> [ERROR] [1590780431.602124884]: No techniques available for material [PointCloudMaterial5FlatSquare]
> [ERROR] [1590780432.113293590]: No techniques available for material [PointCloudMaterial6Sphere]
> [ERROR] [1590780432.113444678]: No techniques available for material [PointCloudMaterial6FlatSquare]
> [ERROR] [1590780432.593522391]: No techniques available for material [PointCloudMaterial7Sphere]
> [ERROR] [1590780432.593660312]: No techniques available for material [PointCloudMaterial7FlatSquare]
> [ERROR] [1590780433.106866094]: No techniques available for material [PointCloudMaterial8Sphere]
> [ERROR] [1590780433.107107751]: No techniques available for material [PointCloudMaterial8FlatSquare]
> [ERROR] [1590780433.599885477]: No techniques available for material [PointCloudMaterial9Sphere]
> [ERROR] [1590780433.599973180]: No techniques available for material [PointCloudMaterial9FlatSquare]
> [ERROR] [1590780434.097377403]: No techniques available for material [PointCloudMaterial10Sphere]
> [ERROR] [1590780434.097551561]: No techniques available for material [PointCloudMaterial10FlatSquare]
> Segmentation fault (core dumped)
>

gave it a go with ogre 1.12....

```
rosrun rviz rviz
[ INFO] [1590852307.372457308]: rviz version 1.14.0
[ INFO] [1590852307.372531006]: compiled against Qt version 5.12.8
[ INFO] [1590852307.372545986]: compiled against OGRE version 1.12.4 (Rhagorthua)
[ INFO] [1590852307.388516299]: Forcing OpenGl version 0.
[ INFO] [1590852307.595392886]: Stereo is NOT SUPPORTED
[ INFO] [1590852307.595582330]: OpenGl version: 4.6 (GLSL 4.6).
[ERROR] [1590852307.607582822]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
[ERROR] [1590852326.258043428]: No techniques available for material [PointCloudMaterial0Sphere]
[ERROR] [1590852326.258421743]: No techniques available for material [PointCloudMaterial0FlatSquare]
[ERROR] [1590852327.280323290]: No techniques available for material [PointCloudMaterial1Sphere]
[ERROR] [1590852327.280439542]: No techniques available for material [PointCloudMaterial1FlatSquare]
[ERROR] [1590852328.338720758]: No techniques available for material [PointCloudMaterial2Sphere]
[ERROR] [1590852328.338899048]: No techniques available for material [PointCloudMaterial2FlatSquare]
[ERROR] [1590852329.391346141]: No techniques available for material [PointCloudMaterial3Sphere]
[ERROR] [1590852329.391653767]: No techniques available for material [PointCloudMaterial3FlatSquare]
```

when running catkin_make with the args you wanted me to there seems to be some error with regards to Vector3 in OGRE.

**catkin_make with args:**
`./src/catkin/bin/catkin_make -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_FLAGS="-fsanitize=address -fno-omit-frame-pointer"`


```bash
In file included from /home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/UVLADIE3JM/moc_plant_flag_tool.cpp:9,
                 from /home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/mocs_compilation.cpp:4:
/home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/UVLADIE3JM/../../../../../src/visualization_tutorials/rviz_plugin_tutorials/src/plant_flag_tool.h:37:7: error: using typedef-name ‘Ogre::Vector3’ after ‘class’
   37 | class Vector3;
      |       ^~~~~~~
In file included from /usr/include/OGRE/OgreVector.h:32,
                 from /usr/include/OGRE/OgreVector3.h:1,
                 from /home/flynn/dev/ros/noetic/src/rviz/src/rviz/frame_manager.h:41,
                 from /home/flynn/dev/ros/noetic/src/rviz/src/rviz/display_context.h:37,
                 from /home/flynn/dev/ros/noetic/src/rviz/src/rviz/message_filter_display.h:37,
                 from /home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/UVLADIE3JM/../../../../../src/visualization_tutorials/rviz_plugin_tutorials/src/imu_display.h:36,
                 from /home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/UVLADIE3JM/moc_imu_display.cpp:9,
                 from /home/flynn/dev/ros/noetic/build/visualization_tutorials/rviz_plugin_tutorials/rviz_plugin_tutorials_autogen/mocs_compilation.cpp:3:
/usr/include/OGRE/OgrePrerequisites.h:253:29: note: ‘Ogre::Vector3’ has a previous declaration here
  253 |     typedef Vector<3, Real> Vector3;
      |                             ^~~~~~~
```

tried to compile with this command according to the arguments to test above: 
`./src/catkin/bin/catkin_make --only-pkg-with-deps rviz -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_FLAGS="-fsanitize=address -fno-omit-frame-pointer"
`

when I go and launch rviz....I get this & rviz crashes **only** when I **move** my mouse through the 3D view within rviz. moving the mouse and interacting with the rest of rviz works great.

<details><summary>rosrun rviz rviz</summary>
<p>

```
[ INFO] [1590975269.466012303]: rviz version 1.14.0
[ INFO] [1590975269.466104596]: compiled against Qt version 5.12.8
[ INFO] [1590975269.466131317]: compiled against OGRE version 1.12.4 (Rhagorthua)
[ INFO] [1590975269.481843791]: Forcing OpenGl version 0.
[ INFO] [1590975269.750340397]: Stereo is NOT SUPPORTED
[ INFO] [1590975269.750467646]: OpenGl version: 4.6 (GLSL 4.6).
[ERROR] [1590975269.783444875]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
=================================================================
==263546==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x6020002395d0 at pc 0x7fd1ac66dfdd bp 0x7ffef5da8e00 sp 0x7ffef5da8df0
READ of size 4 at 0x6020002395d0 thread T0
    #0 0x7fd1ac66dfdc in rviz::SelectionManager::unpackColors(Ogre::PixelBox const&, std::vector<unsigned int, std::allocator<unsigned int> >&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:593
    #1 0x7fd1ac67cec0 in rviz::SelectionManager::renderAndUnpack(Ogre::Viewport*, unsigned int, int, int, int, int, std::vector<unsigned int, std::allocator<unsigned int> >&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:621
    #2 0x7fd1ac67dfba in rviz::SelectionManager::pick(Ogre::Viewport*, int, int, int, int, boost::unordered::unordered_map<unsigned int, rviz::Picked, boost::hash<unsigned int>, std::equal_to<unsigned int>, std::allocator<std::pair<unsigned int const, rviz::Picked> > >&, bool) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:894
    #3 0x7fd18bb153d6 in rviz::InteractionTool::updateFocus(rviz::ViewportMouseEvent const&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/default_plugin/tools/interaction_tool.cpp:85
    #4 0x7fd18bb175cd in rviz::InteractionTool::processMouseEvent(rviz::ViewportMouseEvent&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/default_plugin/tools/interaction_tool.cpp:154
    #5 0x7fd1ac7572ef in rviz::VisualizationManager::handleMouseEvent(rviz::ViewportMouseEvent const&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/visualization_manager.cpp:539
    #6 0x7fd1ac6165df in rviz::RenderPanel::onRenderWindowMouseEvents(QMouseEvent*) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/render_panel.cpp:119
    #7 0x7fd1abd432b5 in QWidget::event(QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1ad2b5)
    #8 0x7fd1abd00a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #9 0x7fd1abd0a342 in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x174342)
    #10 0x7fd1ab8d3939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #11 0x7fd1abd09456 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x173456)
    #12 0x7fd1abd5f35c  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1c935c)
    #13 0x7fd1abd621eb  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1cc1eb)
    #14 0x7fd1abd00a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #15 0x7fd1abd0a0ef in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1740ef)
    #16 0x7fd1ab8d3939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #17 0x7fd1a8cbe7d2 in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x1267d2)
    #18 0x7fd1a8cc010a in QGuiApplicationPrivate::processWindowSystemEvent(QWindowSystemInterfacePrivate::WindowSystemEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x12810a)
    #19 0x7fd1a8c9a35a in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x10235a)
    #20 0x7fd1a00b932d  (/lib/x86_64-linux-gnu/libQt5XcbQpa.so.5+0x7932d)
    #21 0x7fd1a8524fbc in g_main_context_dispatch (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x51fbc)
    #22 0x7fd1a852523f  (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x5223f)
    #23 0x7fd1a85252e2 in g_main_context_iteration (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x522e2)
    #24 0x7fd1ab92b564 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2de564)
    #25 0x7fd1ab8d24da in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2854da)
    #26 0x7fd1ab8da245 in QCoreApplication::exec() (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x28d245)
    #27 0x55679bbd099e in main /home/flynn/dev/ros/noetic/src/rviz/src/rviz/main.cpp:42
    #28 0x7fd1ab2840b2 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x270b2)
    #29 0x55679bbd0a7d in _start (/home/flynn/dev/ros/noetic/devel/lib/rviz/rviz+0x4a7d)

0x6020002395d3 is located 0 bytes to the right of 3-byte region [0x6020002395d0,0x6020002395d3)
allocated by thread T0 here:
    #0 0x7fd1ac9bbb47 in operator new[](unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10fb47)
    #1 0x7fd1ac6781c5 in rviz::SelectionManager::render(Ogre::Viewport*, Ogre::SharedPtr<Ogre::Texture> const&, int, int, int, int, Ogre::PixelBox&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int, unsigned int) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:777
    #2 0x7fd1ac67ce57 in rviz::SelectionManager::renderAndUnpack(Ogre::Viewport*, unsigned int, int, int, int, int, std::vector<unsigned int, std::allocator<unsigned int> >&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:618
    #3 0x7fd1ac67dfba in rviz::SelectionManager::pick(Ogre::Viewport*, int, int, int, int, boost::unordered::unordered_map<unsigned int, rviz::Picked, boost::hash<unsigned int>, std::equal_to<unsigned int>, std::allocator<std::pair<unsigned int const, rviz::Picked> > >&, bool) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:894
    #4 0x7fd18bb153d6 in rviz::InteractionTool::updateFocus(rviz::ViewportMouseEvent const&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/default_plugin/tools/interaction_tool.cpp:85
    #5 0x7fd18bb175cd in rviz::InteractionTool::processMouseEvent(rviz::ViewportMouseEvent&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/default_plugin/tools/interaction_tool.cpp:154
    #6 0x7fd1ac7572ef in rviz::VisualizationManager::handleMouseEvent(rviz::ViewportMouseEvent const&) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/visualization_manager.cpp:539
    #7 0x7fd1ac6165df in rviz::RenderPanel::onRenderWindowMouseEvents(QMouseEvent*) /home/flynn/dev/ros/noetic/src/rviz/src/rviz/render_panel.cpp:119
    #8 0x7fd1abd432b5 in QWidget::event(QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1ad2b5)

SUMMARY: AddressSanitizer: heap-buffer-overflow /home/flynn/dev/ros/noetic/src/rviz/src/rviz/selection/selection_manager.cpp:593 in rviz::SelectionManager::unpackColors(Ogre::PixelBox const&, std::vector<unsigned int, std::allocator<unsigned int> >&)
Shadow bytes around the buggy address:
  0x0c048003f260: fa fa fd fa fa fa fd fd fa fa fd fa fa fa fd fa
  0x0c048003f270: fa fa fd fa fa fa fd fd fa fa fd fd fa fa 00 fa
  0x0c048003f280: fa fa 00 fa fa fa 00 fa fa fa 00 fa fa fa fd fd
  0x0c048003f290: fa fa fd fd fa fa fd fd fa fa 00 fa fa fa 00 00
  0x0c048003f2a0: fa fa 00 00 fa fa fd fa fa fa fd fa fa fa 00 fa
=>0x0c048003f2b0: fa fa fd fa fa fa fd fa fa fa[03]fa fa fa 00 fa
  0x0c048003f2c0: fa fa 04 fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048003f2d0: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048003f2e0: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048003f2f0: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048003f300: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
Shadow byte legend (one shadow byte represents 8 application bytes):
  Addressable:           00
  Partially addressable: 01 02 03 04 05 06 07 
  Heap left redzone:       fa
  Freed heap region:       fd
  Stack left redzone:      f1
  Stack mid redzone:       f2
  Stack right redzone:     f3
  Stack after return:      f5
  Stack use after scope:   f8
  Global redzone:          f9
  Global init order:       f6
  Poisoned by user:        f7
  Container overflow:      fc
  Array cookie:            ac
  Intra object redzone:    bb
  ASan internal:           fe
  Left alloca redzone:     ca
  Right alloca redzone:    cb
  Shadow gap:              cc
==263546==ABORTING
```
</p>
</details

now with ogre 1.12

<details><summary>rosrun rviz rviz</summary>
<p>

```
[ INFO] [1592019381.969945666]: rviz version 1.14.0
[ INFO] [1592019381.970046262]: compiled against Qt version 5.12.8
[ INFO] [1592019381.970070668]: compiled against OGRE version 1.12.4 (Rhagorthua)
[ INFO] [1592019381.995239325]: Forcing OpenGl version 0.
[ INFO] [1592019382.262089622]: Stereo is NOT SUPPORTED
[ INFO] [1592019382.262395223]: OpenGl version: 4.6 (GLSL 4.6).
[ERROR] [1592019382.285713518]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
[ERROR] [1592019422.440086804]: No techniques available for material [PointCloudMaterial0Sphere]
[ERROR] [1592019422.440461544]: No techniques available for material [PointCloudMaterial0FlatSquare]
[ERROR] [1592019422.508575093]: No techniques available for material [PointCloudMaterial1Sphere]
[ERROR] [1592019422.508991574]: No techniques available for material [PointCloudMaterial1FlatSquare]
[ERROR] [1592019422.602285476]: No techniques available for material [PointCloudMaterial2Sphere]
[ERROR] [1592019422.602702484]: No techniques available for material [PointCloudMaterial2FlatSquare]
=================================================================
==2877==ERROR: AddressSanitizer: heap-buffer-overflow on address 0x6020004921b0 at pc 0x7f695cde9bf7 bp 0x7ffdd5926400 sp 0x7ffdd59263f0
READ of size 4 at 0x6020004921b0 thread T0
    #0 0x7f695cde9bf6 in rviz::SelectionManager::unpackColors(Ogre::PixelBox const&, std::vector<unsigned int, std::allocator<unsigned int> >&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5cfbf6)
    #1 0x7f695cdea235 in rviz::SelectionManager::renderAndUnpack(Ogre::Viewport*, unsigned int, int, int, int, int, std::vector<unsigned int, std::allocator<unsigned int> >&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d0235)
    #2 0x7f695cdeda10 in rviz::SelectionManager::pick(Ogre::Viewport*, int, int, int, int, boost::unordered::unordered_map<unsigned int, rviz::Picked, boost::hash<unsigned int>, std::equal_to<unsigned int>, std::allocator<std::pair<unsigned int const, rviz::Picked> > >&, bool) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d3a10)
    #3 0x7f693b584420 in rviz::InteractionTool::updateFocus(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c1420)
    #4 0x7f693b58535f in rviz::InteractionTool::processMouseEvent(rviz::ViewportMouseEvent&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c235f)
    #5 0x7f695ceaced5 in rviz::VisualizationManager::handleMouseEvent(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x692ed5)
    #6 0x7f695cd93910 in rviz::RenderPanel::onRenderWindowMouseEvents(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x579910)
    #7 0x7f695cb93720 in rviz::RenderPanel::mouseMoveEvent(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x379720)
    #8 0x7f695c32c2b5 in QWidget::event(QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1ad2b5)
    #9 0x7f695c2e9a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #10 0x7f695c2f3342 in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x174342)
    #11 0x7f695bebc939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #12 0x7f695c2f2456 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x173456)
    #13 0x7f695c34835c  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1c935c)
    #14 0x7f695c34b1eb  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1cc1eb)
    #15 0x7f695c2e9a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #16 0x7f695c2f30ef in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1740ef)
    #17 0x7f695bebc939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #18 0x7f6958cb07d2 in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x1267d2)
    #19 0x7f6958cb210a in QGuiApplicationPrivate::processWindowSystemEvent(QWindowSystemInterfacePrivate::WindowSystemEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x12810a)
    #20 0x7f6958c8c35a in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x10235a)
    #21 0x7f694ffb932d  (/lib/x86_64-linux-gnu/libQt5XcbQpa.so.5+0x7932d)
    #22 0x7f6958516fbc in g_main_context_dispatch (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x51fbc)
    #23 0x7f695851723f  (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x5223f)
    #24 0x7f69585172e2 in g_main_context_iteration (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x522e2)
    #25 0x7f695bf14564 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2de564)
    #26 0x7f695bebb4da in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2854da)
    #27 0x7f695bec3245 in QCoreApplication::exec() (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x28d245)
    #28 0x5605a4ec79ff in main (/home/flynn/dev/ros/noetic/devel/lib/rviz/rviz+0xc9ff)
    #29 0x7f695b86d0b2 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x270b2)
    #30 0x5605a4ec77cd in _start (/home/flynn/dev/ros/noetic/devel/lib/rviz/rviz+0xc7cd)

0x6020004921b3 is located 0 bytes to the right of 3-byte region [0x6020004921b0,0x6020004921b3)
allocated by thread T0 here:
    #0 0x7f695d1f5b47 in operator new[](unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10fb47)
    #1 0x7f695cdebde1 in rviz::SelectionManager::render(Ogre::Viewport*, Ogre::SharedPtr<Ogre::Texture> const&, int, int, int, int, Ogre::PixelBox&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int, unsigned int) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d1de1)
    #2 0x7f695cdea1c4 in rviz::SelectionManager::renderAndUnpack(Ogre::Viewport*, unsigned int, int, int, int, int, std::vector<unsigned int, std::allocator<unsigned int> >&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d01c4)
    #3 0x7f695cdeda10 in rviz::SelectionManager::pick(Ogre::Viewport*, int, int, int, int, boost::unordered::unordered_map<unsigned int, rviz::Picked, boost::hash<unsigned int>, std::equal_to<unsigned int>, std::allocator<std::pair<unsigned int const, rviz::Picked> > >&, bool) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d3a10)
    #4 0x7f693b584420 in rviz::InteractionTool::updateFocus(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c1420)
    #5 0x7f693b58535f in rviz::InteractionTool::processMouseEvent(rviz::ViewportMouseEvent&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c235f)
    #6 0x7f695ceaced5 in rviz::VisualizationManager::handleMouseEvent(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x692ed5)
    #7 0x7f695cd93910 in rviz::RenderPanel::onRenderWindowMouseEvents(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x579910)
    #8 0x7f695cb93720 in rviz::RenderPanel::mouseMoveEvent(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x379720)
    #9 0x7f695c32c2b5 in QWidget::event(QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1ad2b5)

SUMMARY: AddressSanitizer: heap-buffer-overflow (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5cfbf6) in rviz::SelectionManager::unpackColors(Ogre::PixelBox const&, std::vector<unsigned int, std::allocator<unsigned int> >&)
Shadow bytes around the buggy address:
  0x0c048008a3e0: fa fa fd fd fa fa fd fa fa fa fd fa fa fa fd fa
  0x0c048008a3f0: fa fa fd fd fa fa fd fd fa fa 00 fa fa fa 00 fa
  0x0c048008a400: fa fa 00 fa fa fa 00 fa fa fa fd fd fa fa fd fd
  0x0c048008a410: fa fa fd fd fa fa 00 fa fa fa 00 00 fa fa 00 00
  0x0c048008a420: fa fa fd fa fa fa fd fa fa fa 00 fa fa fa fd fa
=>0x0c048008a430: fa fa fd fa fa fa[03]fa fa fa 00 fa fa fa 04 fa
  0x0c048008a440: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048008a450: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048008a460: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048008a470: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
  0x0c048008a480: fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa fa
Shadow byte legend (one shadow byte represents 8 application bytes):
  Addressable:           00
  Partially addressable: 01 02 03 04 05 06 07 
  Heap left redzone:       fa
  Freed heap region:       fd
  Stack left redzone:      f1
  Stack mid redzone:       f2
  Stack right redzone:     f3
  Stack after return:      f5
  Stack use after scope:   f8
  Global redzone:          f9
  Global init order:       f6
  Poisoned by user:        f7
  Container overflow:      fc
  Array cookie:            ac
  Intra object redzone:    bb
  ASan internal:           fe
  Left alloca redzone:     ca
  Right alloca redzone:    cb
  Shadow gap:              cc
==2877==ABORTING
```
</p>
</details>

symptoms observed:
- crashes immediately if cursor moves within 3d view panel
- only displays pointcloud if 'points' is selected
- cannot resize points if rviz is autosized to fit screen

that PR is into/from melodic-devel which doesnt include the changes you've made on  noetic-devel that allows for compilation with ogre 1.12.

I get the following error when trying to compile those new changes with ogre 1.12:

```
/home/flynn/dev/ros/noetic/src/rviz/src/rviz/view_controller.h:46:7: error: using typedef-name ‘Ogre::Vector3’ after ‘class’
   46 | class Vector3;
      |       ^~~~~~~
In file included from /usr/include/OGRE/OgreVector.h:32,
                 from /usr/include/OGRE/OgreVector3.h:1,
                 from /home/flynn/dev/ros/noetic/src/rviz/src/rviz/frame_manager.h:39,
                 from /home/flynn/dev/ros/noetic/build/rviz/src/rviz/rviz_autogen/EWIEGA46WW/moc_frame_manager.cpp:9,
                 from /home/flynn/dev/ros/noetic/build/rviz/src/rviz/rviz_autogen/mocs_compilation.cpp:8:
/usr/include/OGRE/OgrePrerequisites.h:253:29: note: ‘Ogre::Vector3’ has a previous declaration here
  253 |     typedef Vector<3, Real> Vector3;
      |                             
```

you were right I didnt have those changes included yet.
I tried to just clone the pr but realized it was for melodic and not noetic after a few compilation errors.
manually merged the new commits into noetic-devel.

<details><summary>git log -2 of my current rviz source</summary>
<p>

```
git log -2
commit debf9203e8ecc64f330abfffac985bb6ad8ed08e (HEAD -> noetic-devel)
Merge: c731eff1 6b254a67
Author: Evan Flynn <evanflynn.msu@gmail.com>
Date:   Sat Jun 13 06:59:03 2020 -0700

    Merge commit 'refs/pull/1519/head' of github.com:ros-visualization/rviz into noetic-devel

commit 6b254a67282a05b8cf05b320ee80a58f224b2364 (fix-memory-access)
Author: Robert Haschke <rhaschke@techfak.uni-bielefeld.de>
Date:   Sat Jun 13 07:40:58 2020 +0200

    change private API instead of const-casting
```

<p>
</details>

after compilation....

<details><summary>rosrun rviz rviz</summary>
<p>

```
rosrun rviz rviz
[ INFO] [1592057674.377568903]: rviz version 1.14.0
[ INFO] [1592057674.377715849]: compiled against Qt version 5.12.8
[ INFO] [1592057674.377768214]: compiled against OGRE version 1.12.4 (Rhagorthua)
[ INFO] [1592057674.407054089]: Forcing OpenGl version 0.
[ INFO] [1592057674.661664123]: Stereo is NOT SUPPORTED
[ INFO] [1592057674.661984415]: OpenGl version: 4.6 (GLSL 4.6).
[ERROR] [1592057674.687852394]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
[ERROR] [1592057740.081238943]: No techniques available for material [PointCloudMaterial0Sphere]
[ERROR] [1592057740.081565951]: No techniques available for material [PointCloudMaterial0FlatSquare]
[ERROR] [1592057740.138587982]: No techniques available for material [PointCloudMaterial1Sphere]
[ERROR] [1592057740.138724765]: No techniques available for material [PointCloudMaterial1FlatSquare]
AddressSanitizer:DEADLYSIGNAL
=================================================================
==27135==ERROR: AddressSanitizer: SEGV on unknown address 0x000000000040 (pc 0x7f97b539aa04 bp 0x7ffd4778b3d0 sp 0x7ffd4778b2b8 T0)
==27135==The signal is caused by a READ memory access.
==27135==Hint: address points to the zero page.
    #0 0x7f97b539aa03 in Ogre::UserObjectBindings::getUserAny(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x3ada03)
    #1 0x7f97b7e4ca5b in rviz::SelectionManager::handleSchemeNotFound(unsigned short, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, Ogre::Material*, unsigned short, Ogre::Renderable const*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d5a5b)
    #2 0x7f97b521c157 in Ogre::MaterialManager::_arbitrateMissingTechniqueForActiveScheme(Ogre::Material*, unsigned short, Ogre::Renderable const*) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x22f157)
    #3 0x7f97b5217ec5 in Ogre::Material::getBestTechnique(unsigned short, Ogre::Renderable const*) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x22aec5)
    #4 0x7f97b529bc16 in Ogre::RenderQueue::addRenderable(Ogre::Renderable*, unsigned char, unsigned short) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2aec16)
    #5 0x7f97b7d70d6e in rviz::PointCloud::_updateRenderQueue(Ogre::RenderQueue*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x4f9d6e)
    #6 0x7f97b529b6bb in Ogre::RenderQueue::processVisibleObject(Ogre::MovableObject*, Ogre::Camera*, bool, Ogre::VisibleObjectsBoundsInfo*) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2ae6bb)
    #7 0x7f97b52fa9a8 in Ogre::SceneNode::_findVisibleObjects(Ogre::Camera*, Ogre::RenderQueue*, Ogre::VisibleObjectsBoundsInfo*, bool, bool, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x30d9a8)
    #8 0x7f97b52fa9fe in Ogre::SceneNode::_findVisibleObjects(Ogre::Camera*, Ogre::RenderQueue*, Ogre::VisibleObjectsBoundsInfo*, bool, bool, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x30d9fe)
    #9 0x7f97b52fa9fe in Ogre::SceneNode::_findVisibleObjects(Ogre::Camera*, Ogre::RenderQueue*, Ogre::VisibleObjectsBoundsInfo*, bool, bool, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x30d9fe)
    #10 0x7f97b52e0cf6 in Ogre::SceneManager::_findVisibleObjects(Ogre::Camera*, Ogre::VisibleObjectsBoundsInfo*, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2f3cf6)
    #11 0x7f97b52e9685 in Ogre::SceneManager::_renderScene(Ogre::Camera*, Ogre::Viewport*, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2fc685)
    #12 0x7f97b515d9f9 in Ogre::Camera::_renderScene(Ogre::Viewport*, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x1709f9)
    #13 0x7f97b52b70e4 in Ogre::RenderTarget::_updateViewport(Ogre::Viewport*, bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2ca0e4)
    #14 0x7f97b52b6e91 in Ogre::RenderTarget::_updateAutoUpdatedViewports(bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2c9e91)
    #15 0x7f97b52b6cc2 in Ogre::RenderTarget::updateImpl() (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2c9cc2)
    #16 0x7f97b52b7487 in Ogre::RenderTarget::update(bool) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2ca487)
    #17 0x7f97b7e48c05 in rviz::SelectionManager::render(Ogre::Viewport*, Ogre::SharedPtr<Ogre::Texture> const&, int, int, int, int, Ogre::PixelBox&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, unsigned int, unsigned int) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d1c05)
    #18 0x7f97b7e473d2 in rviz::SelectionManager::renderAndUnpack(Ogre::Viewport*, unsigned int, int, int, int, int, std::vector<unsigned int, std::allocator<unsigned int> >&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d03d2)
    #19 0x7f97b7e4ac1e in rviz::SelectionManager::pick(Ogre::Viewport*, int, int, int, int, boost::unordered::unordered_map<unsigned int, rviz::Picked, boost::hash<unsigned int>, std::equal_to<unsigned int>, std::allocator<std::pair<unsigned int const, rviz::Picked> > >&, bool) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x5d3c1e)
    #20 0x7f97965e1566 in rviz::InteractionTool::updateFocus(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c1566)
    #21 0x7f97965e24a5 in rviz::InteractionTool::processMouseEvent(rviz::ViewportMouseEvent&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c24a5)
    #22 0x7f97b7f0a98d in rviz::VisualizationManager::handleMouseEvent(rviz::ViewportMouseEvent const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x69398d)
    #23 0x7f97b7df098e in rviz::RenderPanel::onRenderWindowMouseEvents(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x57998e)
    #24 0x7f97b7bf0780 in rviz::RenderPanel::mouseMoveEvent(QMouseEvent*) (/home/flynn/dev/ros/noetic/devel/lib/librviz.so+0x379780)
    #25 0x7f97b73892b5 in QWidget::event(QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1ad2b5)
    #26 0x7f97b7346a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #27 0x7f97b7350342 in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x174342)
    #28 0x7f97b6f19939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #29 0x7f97b734f456 in QApplicationPrivate::sendMouseEvent(QWidget*, QMouseEvent*, QWidget*, QWidget*, QWidget**, QPointer<QWidget>&, bool, bool) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x173456)
    #30 0x7f97b73a535c  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1c935c)
    #31 0x7f97b73a81eb  (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1cc1eb)
    #32 0x7f97b7346a65 in QApplicationPrivate::notify_helper(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x16aa65)
    #33 0x7f97b73500ef in QApplication::notify(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Widgets.so.5+0x1740ef)
    #34 0x7f97b6f19939 in QCoreApplication::notifyInternal2(QObject*, QEvent*) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x286939)
    #35 0x7f97b3d0d7d2 in QGuiApplicationPrivate::processMouseEvent(QWindowSystemInterfacePrivate::MouseEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x1267d2)
    #36 0x7f97b3d0f10a in QGuiApplicationPrivate::processWindowSystemEvent(QWindowSystemInterfacePrivate::WindowSystemEvent*) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x12810a)
    #37 0x7f97b3ce935a in QWindowSystemInterface::sendWindowSystemEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Gui.so.5+0x10235a)
    #38 0x7f97ab0b932d  (/lib/x86_64-linux-gnu/libQt5XcbQpa.so.5+0x7932d)
    #39 0x7f97b3573fbc in g_main_context_dispatch (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x51fbc)
    #40 0x7f97b357423f  (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x5223f)
    #41 0x7f97b35742e2 in g_main_context_iteration (/lib/x86_64-linux-gnu/libglib-2.0.so.0+0x522e2)
    #42 0x7f97b6f71564 in QEventDispatcherGlib::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2de564)
    #43 0x7f97b6f184da in QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x2854da)
    #44 0x7f97b6f20245 in QCoreApplication::exec() (/lib/x86_64-linux-gnu/libQt5Core.so.5+0x28d245)
    #45 0x5598361749ff in main (/home/flynn/dev/ros/noetic/devel/lib/rviz/rviz+0xc9ff)
    #46 0x7f97b68ca0b2 in __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x270b2)
    #47 0x5598361747cd in _start (/home/flynn/dev/ros/noetic/devel/lib/rviz/rviz+0xc7cd)

AddressSanitizer can not provide additional info.
SUMMARY: AddressSanitizer: SEGV (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x3ada03) in Ogre::UserObjectBindings::getUserAny(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) const
==27135==ABORTING
```
</p>
</details>

symptoms observed:
- can now move mouse through 3d window without crash
- when a pointcloud topic is added to visualize, rviz crashes immediately

when running with either `--opengl 300` or `--opengl 210` :
- when pointcloud topic is added to visualize, rviz *does not* crash!
- can change size of any points type when rviz is not maximized to fit window
- when maximized to fit window, only 'points' type is not able to change point size.

<details><summary>rosrun rviz rviz --opengl 300</summary>
<p>

```
[ INFO] [1592058924.569122316]: rviz version 1.14.0
[ INFO] [1592058924.569231356]: compiled against Qt version 5.12.8
[ INFO] [1592058924.569276001]: compiled against OGRE version 1.12.4 (Rhagorthua)
[ INFO] [1592058924.599036252]: Forcing OpenGl version 3.
[ INFO] [1592058924.828757378]: Stereo is NOT SUPPORTED
[ INFO] [1592058924.828992698]: OpenGl version: 3 (GLSL 1.3).
[ERROR] [1592058924.850943315]: Error: ScriptCompiler - unexpected token in liberation_sans.fontdef(1): 'Liberation'. If this is a legacy script you must prepend the type (e.g. font, overlay).
^C
=================================================================
==27584==ERROR: LeakSanitizer: detected memory leaks

Direct leak of 4416 byte(s) in 92 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a56044e6 in Ogre::FreeImageCodec::startup() (/usr/lib/x86_64-linux-gnu/OGRE-1.12/Codec_FreeImage.so.1.12.4+0x64e6)
    #2 0x7f32c0525aad in Ogre::Root::loadPlugin(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x2eaaad)
    #3 0x6e6967756c4f  (<unknown module>)

Direct leak of 1200 byte(s) in 30 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32c03ce6b9 in Ogre::ConvexBody::_initialisePool() (/lib/x86_64-linux-gnu/libOgreMain.so.1.12.4+0x1936b9)

Direct leak of 797 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c349fbc8 in malloc (/lib/x86_64-linux-gnu/libasan.so.5+0x10dbc8)
    #1 0x7f32af8db187  (<unknown module>)

Direct leak of 640 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c349fffe in __interceptor_realloc (/lib/x86_64-linux-gnu/libasan.so.5+0x10dffe)
    #1 0x7f32c0104490 in XGetVisualInfo (/lib/x86_64-linux-gnu/libX11.so.6+0x3c490)

Direct leak of 256 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c349fbc8 in malloc (/lib/x86_64-linux-gnu/libasan.so.5+0x10dbc8)
    #1 0x7f32b7ada2f4  (/lib/x86_64-linux-gnu/libfontconfig.so.1+0x212f4)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1135b86 in void class_loader::impl::registerPlugin<rviz::DepthCloudDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x10c7b86)
    #2 0x7f32a111d2d0 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x10af2d0)
    #3 0x7f32a111f569 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x10b1569)
    #4 0x7f32a111ff72 in _GLOBAL__sub_I_depth_cloud_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x10b1f72)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a129cb9c in void class_loader::impl::registerPlugin<rviz::FluidPressureDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x122eb9c)
    #2 0x7f32a129af60 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x122cf60)
    #3 0x7f32a129b5d1 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x122d5d1)
    #4 0x7f32a129b7e0 in _GLOBAL__sub_I_fluid_pressure_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x122d7e0)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a13bc5ce in void class_loader::impl::registerPlugin<rviz::LaserScanDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x134e5ce)
    #2 0x7f32a13ba400 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x134c400)
    #3 0x7f32a13baa71 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x134ca71)
    #4 0x7f32a13bac80 in _GLOBAL__sub_I_laser_scan_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x134cc80)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a13179ca in void class_loader::impl::registerPlugin<rviz::IlluminanceDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a99ca)
    #2 0x7f32a1315e7c in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a7e7c)
    #3 0x7f32a13164ed in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a84ed)
    #4 0x7f32a13166fc in _GLOBAL__sub_I_illuminance_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a86fc)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a154824e in void class_loader::impl::registerPlugin<rviz::PointStampedDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x14da24e)
    #2 0x7f32a1545c8e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x14d7c8e)
    #3 0x7f32a154627e in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x14d827e)
    #4 0x7f32a154644e in _GLOBAL__sub_I_point_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x14d844e)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a13138c4 in void class_loader::impl::registerPlugin<rviz::GridDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a58c4)
    #2 0x7f32a1312eec in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a4eec)
    #3 0x7f32a13134dc in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a54dc)
    #4 0x7f32a1313588 in _GLOBAL__sub_I_grid_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12a5588)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a123ffe0 in void class_loader::impl::registerPlugin<rviz::EffortDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x11d1fe0)
    #2 0x7f32a1236b66 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x11c8b66)
    #3 0x7f32a1237156 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x11c9156)
    #4 0x7f32a1237326 in _GLOBAL__sub_I_effort_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x11c9326)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1828044 in void class_loader::impl::registerPlugin<rviz::InitialPoseTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17ba044)
    #2 0x7f32a18257cc in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b77cc)
    #3 0x7f32a1825d93 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b7d93)
    #4 0x7f32a1825e3f in _GLOBAL__sub_I_initial_pose_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b7e3f)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1072eb7 in void class_loader::impl::registerPlugin<rviz::AxesDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1004eb7)
    #2 0x7f32a107215a in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x100415a)
    #3 0x7f32a107274a in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x100474a)
    #4 0x7f32a10727f6 in _GLOBAL__sub_I_axes_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x10047f6)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a185b65d in void class_loader::impl::registerPlugin<rviz::FrameViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17ed65d)
    #2 0x7f32a1857402 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17e9402)
    #3 0x7f32a1857c42 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17e9c42)
    #4 0x7f32a1857cee in _GLOBAL__sub_I_frame_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17e9cee)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a12d6ddc in void class_loader::impl::registerPlugin<rviz::GridCellsDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1268ddc)
    #2 0x7f32a12d498e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x126698e)
    #3 0x7f32a12d4f7e in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1266f7e)
    #4 0x7f32a12d514e in _GLOBAL__sub_I_grid_cells_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x126714e)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a16cd996 in void class_loader::impl::registerPlugin<rviz::PoseDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x165f996)
    #2 0x7f32a16ca22a in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x165c22a)
    #3 0x7f32a16ca81a in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x165c81a)
    #4 0x7f32a16ca9ea in _GLOBAL__sub_I_pose_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x165c9ea)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a14038b7 in void class_loader::impl::registerPlugin<rviz::MapDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13958b7)
    #2 0x7f32a13fff4e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1391f4e)
    #3 0x7f32a140053e in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x139253e)
    #4 0x7f32a14005ea in _GLOBAL__sub_I_map_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13925ea)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1846596 in void class_loader::impl::registerPlugin<rviz::FixedOrientationOrthoViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d8596)
    #2 0x7f32a18459ee in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d79ee)
    #3 0x7f32a1845fde in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d7fde)
    #4 0x7f32a184608a in _GLOBAL__sub_I_fixed_orientation_ortho_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d808a)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1820b14 in void class_loader::impl::registerPlugin<rviz::GoalTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b2b14)
    #2 0x7f32a181dc76 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17afc76)
    #3 0x7f32a181e23d in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b023d)
    #4 0x7f32a181e2e9 in _GLOBAL__sub_I_goal_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17b02e9)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a183e89c in void class_loader::impl::registerPlugin<rviz::XYOrbitViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d089c)
    #2 0x7f32a183ddba in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17cfdba)
    #3 0x7f32a183e3aa in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d03aa)
    #4 0x7f32a183e456 in _GLOBAL__sub_I_xy_orbit_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d0456)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a182cf09 in void class_loader::impl::registerPlugin<rviz::SelectionTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17bef09)
    #2 0x7f32a182bf1c in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17bdf1c)
    #3 0x7f32a182c50c in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17be50c)
    #4 0x7f32a182c5b8 in _GLOBAL__sub_I_selection_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17be5b8)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a178124a in void class_loader::impl::registerPlugin<rviz::RelativeHumidityDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x171324a)
    #2 0x7f32a177f6fc in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17116fc)
    #3 0x7f32a177fd6d in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1711d6d)
    #4 0x7f32a177ff7c in _GLOBAL__sub_I_relative_humidity_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1711f7c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a135f4f3 in void class_loader::impl::registerPlugin<rviz::InteractiveMarkerDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12f14f3)
    #2 0x7f32a1359108 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12eb108)
    #3 0x7f32a13596f8 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12eb6f8)
    #4 0x7f32a13597a4 in _GLOBAL__sub_I_interactive_marker_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12eb7a4)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1084a6e in void class_loader::impl::registerPlugin<rviz::CameraDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1016a6e)
    #2 0x7f32a1080106 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1012106)
    #3 0x7f32a1080798 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1012798)
    #4 0x7f32a1080844 in _GLOBAL__sub_I_camera_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1012844)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a18325db in void class_loader::impl::registerPlugin<rviz::InteractionTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c45db)
    #2 0x7f32a1830c82 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c2c82)
    #3 0x7f32a1831272 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c3272)
    #4 0x7f32a183131e in _GLOBAL__sub_I_interaction_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17c331e)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1640e95 in void class_loader::impl::registerPlugin<rviz::RGB8PCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d2e95)
    #2 0x7f32a163e272 in (anonymous namespace)::ProxyExec3::ProxyExec3() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d0272)
    #3 0x7f32a163f219 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d1219)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a16408b6 in void class_loader::impl::registerPlugin<rviz::IntensityPCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d28b6)
    #2 0x7f32a163de26 in (anonymous namespace)::ProxyExec2::ProxyExec2() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15cfe26)
    #3 0x7f32a163f20d in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d120d)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a15fd2c8 in void class_loader::impl::registerPlugin<rviz::PointCloudDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x158f2c8)
    #2 0x7f32a15fb806 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x158d806)
    #3 0x7f32a15fbe77 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x158de77)
    #4 0x7f32a15fc086 in _GLOBAL__sub_I_point_cloud_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x158e086)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a14b9682 in void class_loader::impl::registerPlugin<rviz::OdometryDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x144b682)
    #2 0x7f32a14b5d6c in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1447d6c)
    #3 0x7f32a14b635c in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x144835c)
    #4 0x7f32a14b652c in _GLOBAL__sub_I_odometry_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x144852c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a18074d9 in void class_loader::impl::registerPlugin<rviz::TFDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17994d9)
    #2 0x7f32a1804b94 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1796b94)
    #3 0x7f32a18051dc in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17971dc)
    #4 0x7f32a1805288 in _GLOBAL__sub_I_tf_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1797288)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a17c121e in void class_loader::impl::registerPlugin<rviz::TemperatureDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x175321e)
    #2 0x7f32a17bf6d0 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17516d0)
    #3 0x7f32a17bfd41 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1751d41)
    #4 0x7f32a17bff50 in _GLOBAL__sub_I_temperature_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1751f50)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1425520 in void class_loader::impl::registerPlugin<rviz::MarkerArrayDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13b7520)
    #2 0x7f32a1424804 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13b6804)
    #3 0x7f32a1424d77 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13b6d77)
    #4 0x7f32a1424dfc in _GLOBAL__sub_I_marker_array_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13b6dfc)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1642032 in void class_loader::impl::registerPlugin<rviz::XYZPCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d4032)
    #2 0x7f32a163ef56 in (anonymous namespace)::ProxyExec6::ProxyExec6() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d0f56)
    #3 0x7f32a163f23d in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d123d)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a163fcf8 in void class_loader::impl::registerPlugin<rviz::AxisColorPCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d1cf8)
    #2 0x7f32a163d58e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15cf58e)
    #3 0x7f32a163f1f5 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d11f5)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a18180a6 in void class_loader::impl::registerPlugin<rviz::PointTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17aa0a6)
    #2 0x7f32a181587e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a787e)
    #3 0x7f32a1815e6e in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a7e6e)
    #4 0x7f32a1815f1a in _GLOBAL__sub_I_point_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a7f1a)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a17bb376 in void class_loader::impl::registerPlugin<rviz::RobotModelDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x174d376)
    #2 0x7f32a17ba62c in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x174c62c)
    #3 0x7f32a17bac1c in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x174cc1c)
    #4 0x7f32a17bacc8 in _GLOBAL__sub_I_robot_model_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x174ccc8)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1689ee0 in void class_loader::impl::registerPlugin<rviz::PoseArrayDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x161bee0)
    #2 0x7f32a1686c96 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1618c96)
    #3 0x7f32a1687286 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1619286)
    #4 0x7f32a1687456 in _GLOBAL__sub_I_pose_array_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1619456)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1583996 in void class_loader::impl::registerPlugin<rviz::PointCloud2Display, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1515996)
    #2 0x7f32a15814aa in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15134aa)
    #3 0x7f32a1581b1b in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1513b1b)
    #4 0x7f32a1581d2a in _GLOBAL__sub_I_point_cloud2_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1513d2a)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1641474 in void class_loader::impl::registerPlugin<rviz::MONO8PCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d3474)
    #2 0x7f32a163e6be in (anonymous namespace)::ProxyExec4::ProxyExec4() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d06be)
    #3 0x7f32a163f225 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d1225)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a184e2d1 in void class_loader::impl::registerPlugin<rviz::FPSViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17e02d1)
    #2 0x7f32a184b0b0 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17dd0b0)
    #3 0x7f32a184bc06 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17ddc06)
    #4 0x7f32a184bceb in _GLOBAL__sub_I_fps_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17ddceb)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a16402d7 in void class_loader::impl::registerPlugin<rviz::FlatColorPCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d22d7)
    #2 0x7f32a163d9da in (anonymous namespace)::ProxyExec1::ProxyExec1() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15cf9da)
    #3 0x7f32a163f201 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d1201)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a17467cb in void class_loader::impl::registerPlugin<rviz::RangeDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x16d87cb)
    #2 0x7f32a1744a00 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x16d6a00)
    #3 0x7f32a1744ff0 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x16d6ff0)
    #4 0x7f32a17451c0 in _GLOBAL__sub_I_range_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x16d71c0)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1641a53 in void class_loader::impl::registerPlugin<rviz::RGBF32PCTransformer, rviz::PointCloudTransformer>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d3a53)
    #2 0x7f32a163eb0a in (anonymous namespace)::ProxyExec5::ProxyExec5() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d0b0a)
    #3 0x7f32a163f231 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d1231)
    #4 0x7f32a163f25c in _GLOBAL__sub_I_point_cloud_transformers.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15d125c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1842d7a in void class_loader::impl::registerPlugin<rviz::ThirdPersonFollowerViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d4d7a)
    #2 0x7f32a1842348 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d4348)
    #3 0x7f32a1842938 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d4938)
    #4 0x7f32a18429e4 in _GLOBAL__sub_I_third_person_follower_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17d49e4)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1812f25 in void class_loader::impl::registerPlugin<rviz::MoveTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a4f25)
    #2 0x7f32a1812574 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a4574)
    #3 0x7f32a1812bd5 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a4bd5)
    #4 0x7f32a1812ca4 in _GLOBAL__sub_I_move_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a4ca4)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1811697 in void class_loader::impl::registerPlugin<rviz::MeasureTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a3697)
    #2 0x7f32a1810daa in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a2daa)
    #3 0x7f32a181139a in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a339a)
    #4 0x7f32a1811446 in _GLOBAL__sub_I_measure_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a3446)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1922e6a in void class_loader::impl::registerPlugin<rviz::WrenchStampedDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x18b4e6a)
    #2 0x7f32a192081e in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x18b281e)
    #3 0x7f32a1920e0e in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x18b2e0e)
    #4 0x7f32a1920fde in _GLOBAL__sub_I_wrench_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x18b2fde)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a164a6a9 in void class_loader::impl::registerPlugin<rviz::PolygonDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15dc6a9)
    #2 0x7f32a164888c in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15da88c)
    #3 0x7f32a1648e7c in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15dae7c)
    #4 0x7f32a164904c in _GLOBAL__sub_I_polygon_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x15db04c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1502499 in void class_loader::impl::registerPlugin<rviz::PathDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1494499)
    #2 0x7f32a14fe9a8 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x14909a8)
    #3 0x7f32a14fef98 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1490f98)
    #4 0x7f32a14ff168 in _GLOBAL__sub_I_path_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x1491168)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a180ee86 in void class_loader::impl::registerPlugin<rviz::FocusTool, rviz::Tool>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a0e86)
    #2 0x7f32a180e4fc in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a04fc)
    #3 0x7f32a180eaec in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a0aec)
    #4 0x7f32a180eb98 in _GLOBAL__sub_I_focus_tool.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17a0b98)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a183a2fc in void class_loader::impl::registerPlugin<rviz::OrbitViewController, rviz::ViewController>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17cc2fc)
    #2 0x7f32a1838fa4 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17cafa4)
    #3 0x7f32a18396c1 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17cb6c1)
    #4 0x7f32a183976d in _GLOBAL__sub_I_orbit_view_controller.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x17cb76d)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a1444330 in void class_loader::impl::registerPlugin<rviz::MarkerDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13d6330)
    #2 0x7f32a143b632 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13cd632)
    #3 0x7f32a143bc22 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13cdc22)
    #4 0x7f32a143bdf2 in _GLOBAL__sub_I_marker_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x13cddf2)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a13527e4 in void class_loader::impl::registerPlugin<rviz::ImageDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12e47e4)
    #2 0x7f32a134f960 in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12e1960)
    #3 0x7f32a1351b79 in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12e3b79)
    #4 0x7f32a135255c in _GLOBAL__sub_I_image_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x12e455c)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 160 byte(s) in 1 object(s) allocated from:
    #0 0x7f32c34a1947 in operator new(unsigned long) (/lib/x86_64-linux-gnu/libasan.so.5+0x10f947)
    #1 0x7f32a170c990 in void class_loader::impl::registerPlugin<rviz::PoseWithCovarianceDisplay, rviz::Display>(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x169e990)
    #2 0x7f32a170845a in (anonymous namespace)::ProxyExec0::ProxyExec0() (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x169a45a)
    #3 0x7f32a1708a4a in __static_initialization_and_destruction_0(int, int) (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x169aa4a)
    #4 0x7f32a1708c1a in _GLOBAL__sub_I_pose_with_covariance_display.cpp (/home/flynn/dev/ros/noetic/devel/lib/librviz_default_plugin.so+0x169ac1a)
    #5 0x7f32c3e11b89  (/lib64/ld-linux-x86-64.so.2+0x11b89)

Direct leak of 64 byte(s) in 2 object(s) allocated from:
    #0 0x7f32c349fbc8 in malloc (/lib/x86_64-linux-gnu/libasan.so.5+0x10dbc8)
    #1 0x7f32b442f3cf in XextAddDisplay (/lib/x86_64-linux-gnu/libXext.so.6+0xe3cf)
```
</p>
</details>

--- a/src/rviz/ogre_helpers/render_system.cpp
+++ b/src/rviz/ogre_helpers/render_system.cpp
@@ -171,6 +171,7 @@ void RenderSystem::loadOgrePlugins()
 
 void RenderSystem::detectGlVersion()
 {
+  bool mesa_workaround = false;
   if (force_gl_version_)
   {
     gl_version_ = force_gl_version_;
@@ -178,11 +179,11 @@ void RenderSystem::detectGlVersion()
   else
   {
     Ogre::RenderSystem* renderSys = ogre_root_->getRenderSystem();
-    renderSys->createRenderSystemCapabilities();
-    const Ogre::RenderSystemCapabilities* caps = renderSys->getCapabilities();
+    const Ogre::RenderSystemCapabilities* caps = renderSys->createRenderSystemCapabilities();
     int major = caps->getDriverVersion().major;
     int minor = caps->getDriverVersion().minor;
     gl_version_ = major * 100 + minor * 10;
+    mesa_workaround = caps->getDeviceName().find("Mesa ") != std::string::npos && gl_version_ >= 320;
   }
 
   switch (gl_version_)
@@ -213,8 +214,16 @@ void RenderSystem::detectGlVersion()
     }
     break;
   }
-  ROS_INFO_STREAM("OpenGl version: " << (float)gl_version_ / 100.0 << " (GLSL "
-                                     << (float)glsl_version_ / 100.0 << ").");
+  if (mesa_workaround)
+  { // https://github.com/ros-visualization/rviz/issues/1508
+    ROS_INFO("OpenGl version: %.1f (GLSL %.1f) limited to GLSL 1.4 on Mesa system.",
+             (float)gl_version_ / 100.0, (float)glsl_version_ / 100.0);
+
+    gl_version_ = 310;
+    glsl_version_ = 140;
+    return;
+  }
+  ROS_INFO("OpenGl version: %.1f (GLSL %.1f).", (float)gl_version_ / 100.0, (float)glsl_version_ / 100.0);
 }
 
 void RenderSystem::setupRenderSystem()

--- a/src/rviz/properties/combo_box.cpp
+++ b/src/rviz/properties/combo_box.cpp
@@ -27,12 +27,17 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "rviz/properties/combo_box.h"
+#include <rviz/properties/combo_box.h>
+#include <QAbstractItemView>
 
 namespace rviz
 {
-ComboBox::ComboBox(QWidget* parent) : QComboBox(parent)
+// allow popup list to become larger than QCombBox itself
+void ComboBox::showPopup()
 {
+  auto view = this->view();
+  view->setMinimumWidth(view->sizeHintForColumn(0));
+  QComboBox::showPopup();
 }
 
 } // end namespace rviz

--- a/src/rviz/properties/combo_box.h
+++ b/src/rviz/properties/combo_box.h
@@ -38,7 +38,8 @@ class ComboBox : public QComboBox
   Q_OBJECT
   Q_PROPERTY(QString currentText READ currentText USER true)
 public:
-  ComboBox(QWidget* parent = nullptr);
+  using QComboBox::QComboBox;
+  void showPopup() override;
 };
 
 } // end namespace rviz

--- a/src/rviz/properties/editable_enum_property.cpp
+++ b/src/rviz/properties/editable_enum_property.cpp
@@ -61,6 +61,8 @@ QWidget* EditableEnumProperty::createEditor(QWidget* parent, const QStyleOptionV
   Q_EMIT requestOptions(this);
 
   EditableComboBox* cb = new EditableComboBox(parent);
+  // avoid larger comboxbox than column width of PropertyWidget
+  cb->setSizeAdjustPolicy(QComboBox::AdjustToMinimumContentsLengthWithIcon);
   cb->addItems(strings_);
   cb->setEditText(getValue().toString());
   QObject::connect(cb, SIGNAL(currentIndexChanged(const QString&)), this,

--- a/src/rviz/properties/splitter_handle.cpp
+++ b/src/rviz/properties/splitter_handle.cpp
@@ -50,7 +50,8 @@ SplitterHandle::SplitterHandle(QTreeView* parent)
 bool SplitterHandle::eventFilter(QObject* event_target, QEvent* event)
 {
   if (event_target == parent_ &&
-      (event->type() == QEvent::Resize || event->type() == QEvent::LayoutRequest))
+      (event->type() == QEvent::Resize || event->type() == QEvent::LayoutRequest ||
+       event->type() == QEvent::Show))
   {
     updateGeometry();
   }

